MagneticField/ParametrizedEngine/interface/alpaka/ParabolicParametrizedMagneticField.h

```diff
@@ -0,0 +1,51 @@
+/**
+ Description: Utility function to calculate the Magnetic Field on the GPU
+*/
+
+#ifndef MagneticField_ParametrizedEngine_interface_alpaka_ParabolicParametrizedMagneticField_h
+#define MagneticField_ParametrizedEngine_interface_alpaka_ParabolicParametrizedMagneticField_h
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  namespace MagneticFieldParabolicPortable {
+
+    struct Parameters {
+      // These parameters are the best fit of 3.8T to the OAEParametrizedMagneticField parametrization.
+      // See MagneticField/ParametrizedEngine/src/ParabolicParametrizedMagneticField.cc
+      static constexpr float c1 = 3.8114;
+      static constexpr float b0 = -3.94991e-06;
+      static constexpr float b1 = 7.53701e-06;
+      static constexpr float a = 2.43878e-11;
+      static constexpr float max_radius2 = 13225.f;  // tracker radius
+      static constexpr float max_z = 280.f;          // tracker z
+    };
+
+    template <typename Vec3>
+    constexpr float Kr(Vec3 const& vec) {
+      return Parameters::a * (vec(0) * vec(0) + vec(1) * vec(1)) + 1.;
```

Comment:
```suggestion
      return Parameters::a * (vec(0) * vec(0) + vec(1) * vec(1)) + 1.f;
```
to a void a conversion to `double` and back to `float`.

Reply:
OK: https://github.com/cms-sw/cmssw/pull/44360/commits/058bf5d544970ce9f1fb9fc228e4da2b1a300cf4